docs/content/Coding Milestones/PoC2023q1/example1-subs/example1-stage-1a.md

```diff
@@ -82,25 +80,23 @@ compute                                                    <none>       mqnl7r5f
 espw                                                       <none>       2n88ugkhysjbxqp5
 imw1                                                       <none>       4d2r9stcyy2qq5c1
 ```
-
-Also: if you ever need to look up just one mailbox workspace by
+  
```

Comment:
Why add spaces here?

Reply:
What should we do in places where the text refers to 'mailbox workspace'?  Change to mailbox space?